Adyen/Core/Components/Base/PresentableComponent.swift

```diff
@@ -43,7 +43,15 @@ public extension PresentableComponent {
 }
 
 /// :nodoc:
-public protocol TrackableComponent: Component, PaymentMethodAware, ViewControllerDelegate {}
+public protocol TrackableComponent: Component, PaymentMethodAware, ViewControllerDelegate {
+
+    /// :nodoc:
+    func viewDidLoad(viewController: UIViewController)
```

Comment:
did you run into the issue where you could not override default implementation of the `ViewControllerDelegate` protocol? :)

Reply:
WIP